tools/cli/package.json

```diff
@@ -25,6 +25,7 @@
     "@lg-tools/test": "0.4.6",
     "@lg-tools/update": "0.1.8",
     "@lg-tools/validate": "0.1.10",
+    "@lg-tools/migrator": "0.0.1",
```

Comment:
Should this be called `migrator`?  Would this be confused with the MongoDB Migrator product? Would `codemods` be a better package name?

Reply:
You might be right, `codemods` would be a better name to avoid confusion

Reply:
I'll bring this up during dev sync!

Reply:
`lg-tools/codemods`
`yarn lg migrate`
`yarn lg codemode`